__tests__/matchers/templates.js

```diff
@@ -0,0 +1,66 @@
+// Goal with these matchers is to make tests more resilient to changes in template model.
+
+expect.extend({
+  toBeSubjectTemplate(subjectTemplate, subjectTemplateKey) {
+    if (typeof subjectTemplateKey !== 'string') {
+      throw new Error('expected subjectTemplateKey to be a string')
+    }
+
+    // Make sure the keys match and this is a subject template (indicated by having class)
```

Comment:
Are you using this check in subsequent testing PRs?

Reply:
That is the hope.